### PR TITLE
Introduction to Git: Remove colon after "Lesson Overview" title

### PR DESCRIPTION
**Pull request by thecodetato**
## Because
Following an issue opened about an out place colon in the Introduction to Git lesson, after Lesson Overview title.

## This PR
- Saw an open issue
- Removed the colon as per the issue
- PR created

## Issue

Closes https://github.com/TheOdinProject/curriculum/issues/25941

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by ManonLef** [comment]
It looks like this PR has conflicts that have to be resolved before we can merge. Could you look into this? It could be that your local clone is not up to date with the curriculum repo from TOP. Which is what I expect since we recently updated all the headers to sentence case and yours seems to still have wiki style headings :)

**Comment by thecodetato** [comment]
@ManonLef Thank you for the feedback! 

I will put this into practice for future ref, and also apologies for any this may have inconvenience caused.

**Comment by thecodetato** [comment]
> It looks like this PR has conflicts that have to be resolved before we can merge. Could you look into this? It could be that your local clone is not up to date with the curriculum repo from TOP. Which is what I expect since we recently updated all the headers to sentence case and yours seems to still have wiki style headings :)

Yes, checking right now!

File: git/foundations_git/introduction_to_git.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ Finally, you will set up a project with Git that will serve as a template for yo
 
 For now, let's learn what Git is and why it's so powerful!
 
-### Lesson overview:
+
+### Lesson Overview
```
Comment (ManonLef) [inline git/foundations_git/introduction_to_git.md:24]:
```suggestion
### Lesson overview
```

We want to keep sentence case according to our style guide. 
I removed the extra line above the header. 

You can click `commit suggestion` to directly commit my suggestion if you want. That way you don't ahve to locally edit and push it again

**Review by ManonLef — CHANGES_REQUESTED** [review]
I left a suggestion below :)

**Comment by ManonLef** [comment]
Perfect! Thanks so much for picking this up!

**Review by ManonLef — APPROVED** [review]

File: git/foundations_git/introduction_to_git.md (hunk 1 above)
Comment (thecodetato) [reply git/foundations_git/introduction_to_git.md]: Thank you so much!

**Pull request merged**